scripts/classes/app-configurator.js

```diff
@@ -70,11 +75,11 @@ class AppConfigurator {
       }, (error, response, body) => {
         if (response.statusCode === 200) {
           const configuration = JSON.parse(body);
-          console.timeEnd('download configuration');
+          console.timeEnd('Download configuration'.bold.green);
           this.configuration = configuration;
           resolve(configuration);
         } else {
-          reject(`Configuration download failed! Error code: ${response.statusCode}`);
+          reject('Configuration download failed!', `Error code: ${response.statusCode}`.bold.red);
```

Comment:
Same as above. We should improve error handling in this script. You can implement this later, in a separate pull request.